lametro/templates/partials/meeting_details_current.html

```diff
@@ -83,46 +83,43 @@ <h4>
                         <i class="fa fa-fw fa-clock-o"></i> {{ current_meeting.first.start_time | date:"g:i a"}}<br/>
                         <i class="fa fa-fw fa-map-marker"></i> {{ current_meeting.first.location.name}}<br />
                     </p>
-
-                    {% if current_meeting.first.documents.all %}
-                        <div class="text-center">
-                            <p>
-                                <a class="btn btn-teal" id="pdf-download-link" target='_blank' href='{{ current_meeting.first.documents.all|find_agenda_url }}'>
-                                    <i class='fa fa-fw fa-download'></i>
-                                    Get Agenda PDF
-                                </a>
-                            </p>
-                        </div>
-                    {% endif %}
                 </div>
-            </div>
 
             <!-- Buttons -->
-            <div class="row">
-                <div class="col-md-12 text-center">
-                    <p>
-                        <a class="btn btn-salmon" href="{{ current_meeting.first.english_live_media_url }}" target="_blank" style="margin-top: 10px; margin-left: 5px;">
-                            <i class="fa fa-headphones" aria-hidden="true"></i>
-                            Watch in English
-                        </a>
-
-                        {% if bilingual %}
-                            <a class="btn btn-salmon" href="{{ current_meeting.first.spanish_live_media_url }}" target="_blank" style="margin-top: 10px; margin-left: 5px;">
-                                <i class="fa fa-headphones" aria-hidden="true"></i>
-                                Ver en Español
-                            </a>
-                        {% endif %}
+            <div class="text-center">
+                <div style="display:inline-block;">
+				  <div class="btn-group btn-group-sm current-meeting-info" role="toolbar">
+					<a class="btn btn-salmon" href="{{ current_meeting.first.english_live_media_url }}" target="_blank" >
+					  <i class="fa fa-headphones" aria-hidden="true"></i>
+					  Watch in English
+					</a>
+
+					{% if bilingual %}
+					<a class="btn btn-salmon" href="{{ current_meeting.first.spanish_live_media_url }}" target="_blank">
+					  <i class="fa fa-headphones" aria-hidden="true"></i>
+					  Ver en Español
+					</a>
+					{% endif %}
+
+				  </div>
+
+				  {% if USING_ECOMMENT %}
+				  <a class="btn btn-sm btn-salmon current-meeting-info" href="{{ current_meeting.first.ecomment_url }}" target="_blank">
+					<i class='fa fa-fw fa-external-link'></i>
+					Go to public comment
+				  </a>
+				  {% endif %}
+
+				  {% if current_meeting.first.documents.all %}
+					<a class="btn btn-sm btn-teal current-meeting-info" id="pdf-download-link" target='_blank' href='{{ current_meeting.first.documents.all|find_agenda_url }}'>
+					  <i class='fa fa-fw fa-download'></i>
+					  Get Agenda PDF
+					</a>
+				  {% endif %}
```

Comment:
Can you fix this indentation so each level is nested one level. Re: 2 or 4 spaces, be consistent with the rest of this file.